src/nvim/path.c

```diff
@@ -563,11 +562,9 @@ bool path_has_exp_wildcard(const char *p)
       p++;
       continue;
     }
+#endif
 
     const char *wildcards = "*?[{";
-#else
-    const char *wildcards = "*?[";  // Windows.
-#endif
```

Comment:
This change makes behavior more consistent, considering the fact that `{` does have a special meaning on Windows when there are with wildcards.